core/src/main/java/jenkins/model/item_category/Category.java

```diff
@@ -0,0 +1,70 @@
+package jenkins.model.item_category;
+
+import hudson.model.TopLevelItem;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents an {@link ItemCategory} and its {@link TopLevelItem}s.
+ *
+ * This class is not thread-safe.
```

Comment:
:ant: again seems like these are not really useful as API classes and could be restricted.